### ISSUE TITLE
XFCE: Change wallpaper on all workspaces and monitors

### ISSUE THREAD

**Issue by Baldomo** [vineetred/flowy#42]
`wallpaper_rs` should change all properties ending with `last-image` in XFCE. As of the latest commit, it only changes wallpaper [in the primary screen and first workspace](https://github.com/vineetred/flowy/blob/55832e26b532359d9a78dde983c658feec6ee01f/wallpaper_rs/src/linux.rs#L75), whereas it should account for multiple workspaces and monitors.

**Describe the solution you'd like**
In bash/command line, the values to change with `xfconf-query` can be found with
```bash
xfconf-query -c xfce4-desktop -p /backdrop/screen0 -lv | grep -E '^\/.+(last-image)\ ' -o
```
<details>
    <summary>Example output</summary>

   <pre>/backdrop/screen0/monitor0/last-image 
/backdrop/screen0/monitor0/workspace0/last-image 
/backdrop/screen0/monitor0/workspace1/last-image 
/backdrop/screen0/monitor0/workspace2/last-image 
/backdrop/screen0/monitor0/workspace3/last-image 
/backdrop/screen0/monitor0/workspace4/last-image 
/backdrop/screen0/monitoreDP-1-1/workspace0/last-image 
/backdrop/screen0/monitoreDP-1-1/workspace1/last-image 
/backdrop/screen0/monitoreDP-1-1/workspace2/last-image 
/backdrop/screen0/monitoreDP-1-1/workspace3/last-image 
/backdrop/screen0/monitoreDP-1-1/workspace4/last-image 
/backdrop/screen0/monitoreDP-1/workspace0/last-image 
/backdrop/screen0/monitoreDP1/workspace0/last-image 
/backdrop/screen0/monitoreDP-1/workspace1/last-image 
/backdrop/screen0/monitoreDP1/workspace1/last-image 
/backdrop/screen0/monitoreDP-1/workspace2/last-image 
/backdrop/screen0/monitoreDP1/workspace2/last-image 
/backdrop/screen0/monitoreDP-1/workspace3/last-image 
/backdrop/screen0/monitoreDP1/workspace3/last-image 
/backdrop/screen0/monitoreDP-1/workspace4/last-image 
/backdrop/screen0/monitoreDP1/workspace4/last-image 
/backdrop/screen0/monitorHDMI-1-2/workspace0/last-image 
/backdrop/screen0/monitorHDMI-1-2/workspace1/last-image 
/backdrop/screen0/monitorHDMI-1-2/workspace2/last-image 
/backdrop/screen0/monitorHDMI-1-2/workspace3/last-image 
/backdrop/screen0/monitorHDMI-2/workspace0/last-image 
/backdrop/screen0/monitorHDMI-2/workspace1/last-image 
/backdrop/screen0/monitorHDMI-2/workspace2/last-image 
/backdrop/screen0/monitorHDMI-2/workspace3/last-image 
/backdrop/screen0/monitorHDMI-2/workspace4/last-image</pre>
</details>

**Describe alternatives you've considered**
There should be an easy enough way to interface with xfconf-query, I was thinking about parsing the output of `xfconf-query -c xfce4-desktop -p /backdrop/screen0 -lv`

**Comment by vineetred** [comment]
Should be an easy fix -- just need to parse all the options into an array and cycle through them. Will get to it over the weekend.

**Comment by Baldomo** [comment]
I already have a prototype working, I could make a PR. I also discovered a bug: after [quoting the path here](https://github.com/vineetred/flowy/blob/55832e26b532359d9a78dde983c658feec6ee01f/wallpaper_rs/src/linux.rs#L38), the string will be formatted as `"1234567path"`, so [slicing it to `[7..]` here](https://github.com/vineetred/flowy/blob/55832e26b532359d9a78dde983c658feec6ee01f/wallpaper_rs/src/linux.rs#L69) yields `path"` (notice the trailing quotation mark). This results in an invalid path passed to `xfconf-query`.

**Comment by vineetred** [comment]
Good catch. Do make a PR then.

**Comment by vineetred** [comment]
Fixed by PR #43
